contrib/opendap/OPeNDAP.md

```diff
@@ -0,0 +1,157 @@
+
+# The OPeNDAP extension to ASPECT
+
+The OPeNDAP client extension to ASPECT has been tested on OSX (10.13,
```

Comment:
I think it would be useful if you started this section with a sentence or two what opendap actually is. People may read this file trying to find out.